.gitignore

```diff
@@ -15,3 +15,5 @@ dev.env
 
 # Ignore frontend build
 internal/frontend/build
+next
+frontend@*
```

Comment:
why `@`?

Reply:
These are generated by next when building the frontend using the make rules